openapi/openapi.yaml

```diff
@@ -83,6 +83,32 @@ paths:
               schema:
                 $ref: "#/components/schemas/APIError"
 
+  /benchmark/plot/{benchmark_id}:
+    get:
+      summary: Returns data needed to generate plots for global benchmark
+      security: []
+      description: Returns data needed to generate plots for global benchmark.
```

Comment:
```suggestion
      summary: Returns data needed to generate plots for a benchmark
      security: []
      description: Returns data needed to generate plots for a benchmark.
```

Reply:
Similar logic to comment 1; I was under the impression that the only time we would need to plot would be for the "global benchmark"(s) because the goal of plotting would be to track the world's collective progress.